static/templates/settings/custom-user-profile-field.handlebars

```diff
@@ -17,6 +17,8 @@
     <input class="custom_user_field_value datepicker" data-field-id="{{ field.id }}" type="text"
       value="{{ field_value.value }}" />
     <span class="remove_date"><i class="fa fa-close"></i></span>
+    {{else if is_url_field }}
+    <textarea maxlength="150" class="custom_user_field_value" type="{{ field_type }}" value="{{ field_value.value }}"></textarea>
```

Comment:
This isn't what I had in mind.  What I meant was for "External account", "custom" type, to make that URL pattern input wider.